cirq/line/line_qubit.py

```diff
@@ -49,3 +62,23 @@ def __repr__(self):
 
     def __str__(self):
         return '{}'.format(self.x)
+
+    def __add__(self, other: int) -> 'LineQubit':
+        assert isinstance(other, int), (
+            'Can only add ints and LineQubits. Instead was {}'.format(other))
+        return LineQubit(self.x + other)
+
+    def __sub__(self, other: int) -> 'LineQubit':
+        assert isinstance(
+            other, int), ('Can only subtract ints and LineQubits. Instead was'
+                          '{}'.format(other))
```

Comment:
I missed this before but all these assertions should probably be TypeErrors.

Reply:
Fixed.  Also missed that the linequbit test imported LineQubit directly so fixed this.  And I was missing a check of the type error in line qubit.